database/rrd.h

```diff
@@ -491,21 +491,18 @@ typedef enum rrdset_flags {
                                                      // No new values have been collected for this chart since agent start, or it was marked RRDSET_FLAG_OBSOLETE at
                                                      // least rrdset_free_obsolete_time seconds ago.
     RRDSET_FLAG_ARCHIVED                = (1 << 15),
-//    RRDSET_FLAG_ACLK                    = (1 << 16), // not used anymore
+    //                                  = (1 << 16),  reserved (not used anymore)
     RRDSET_FLAG_PENDING_FOREACH_ALARMS  = (1 << 17), // contains dims with uninitialized foreach alarms
     RRDSET_FLAG_ANOMALY_DETECTION       = (1 << 18), // flag to identify anomaly detection charts.
     RRDSET_FLAG_INDEXED_ID              = (1 << 19), // the rrdset is indexed by its id
     RRDSET_FLAG_INDEXED_NAME            = (1 << 20), // the rrdset is indexed by its name
-
-    RRDSET_FLAG_ANOMALY_RATE_CHART      = (1 << 21), // the rrdset is for storing anomaly rates for all dimensions
```

Comment:
@stelfrag I think it's safe to simply remove this. What do you think?

Reply:
https://github.com/netdata/netdata/pull/13763#issuecomment-1268581781

i think we should leave this in for now and then remove later when FE ready.

Reply:
fyi - i had them as seperate tasks in this plan: https://github.com/netdata/product/issues/2986

Reply:
@vkalintiris fyi